# Fix the shebang line in a couple of scripts to use a specific version of
# Python so that the pidgin package can be properly published in Userland
# where these things matter.
#
# This patch will not be sent upstream.

--- pidgin-2.12.0/libpurple/purple-remote.orig	2017-03-12 09:00:32.355626039 +0000
+++ pidgin-2.12.0/libpurple/purple-remote	2017-03-12 09:00:43.077940312 +0000
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python2.7
 
 import codecs
 import dbus
--- pidgin-2.12.0/libpurple/purple-url-handler.orig	2017-03-12 08:59:33.344547798 +0000
+++ pidgin-2.12.0/libpurple/purple-url-handler	2017-03-12 08:59:40.209980832 +0000
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python2.7
 from __future__ import print_function
 import dbus
 import re
